package.json

```diff
@@ -111,7 +111,7 @@
     "date-fns-tz": "^2.0.0",
     "dom-serializer": "^0.2.2",
     "domhandler": "^4.3.0",
-    "expensify-common": "git+ssh://git@github.com/Expensify/expensify-common.git#18fa764be9d68f72b48d238dcc20f2b0ca8f1147",
+    "expensify-common": "^2.0.2",
```

Comment:
Why are we switching this from using the hash?

Reply:
After [this](https://github.com/Expensify/expensify-common/pull/699) change, `expensify-common` is published to npm, this was necessary because with TS we have to take build step into consideration.